dockers/actions-plan-preview/main.go

```diff
@@ -72,6 +75,14 @@ func main() {
 		log.Printf("Successfully commented plan-preview result on pull request\n%s\n", *comment.HTMLURL)
 	}
 
+	doMinimizeComment := func(commentID githubv4.ID) {
```

Comment:
Do not define a function at a place too far from where it is being used.
So let's move it to line 132.